lib/pharos/up_command.rb

```diff
@@ -104,6 +104,13 @@ def configure(config, config_content:)
 
       puts pastel.green("==> Starting to craft cluster ...")
       manager.apply_phases
+      if Phase.cluster_context['secrets_encryption']
+        puts pastel.red("==> Configured secrets encryption with keys:")
+        puts pastel.red("    Key1:#{Phase.cluster_context['secrets_encryption']['key1']}")
+        puts pastel.red("    Key2:#{Phase.cluster_context['secrets_encryption']['key2']}")
+        puts pastel.red("    Keep these safe!")
```

Comment:
Doesn't sound reasonable to expect the user to copy-paste these somewhere from the output.

IMO in practice this would be more like: When backing up the kube master node, you must also backup the secrets encryption keys in `/etc/pharos/secrets-encryption/config.yml` in addition to the etcd database in `/var/lib/etcd`.

Reply:
And that advice should probably go into our "Backing up and restoring your kube cluster" docs, rather than into the `pharos-cluster up` output...

Reply:
I agree that output is not the best place for these. Maybe just hint where encryption is stored?

Reply:
Right, I'll remove it completely from the output.

Not sure if it makes sense to "hint" on backing up this as we don't make those hints for anything else.

But in the end, our backup automation should take this in too.